python/sparknlp/internal/py4j_utils.py

```diff
@@ -0,0 +1,32 @@
+# Coloring the print
+FAIL = '\033[91m'
+ENDC = '\033[0m'
+
+
+def check_obj_exist(java_class):
+    # only call this after JVM is turned on
+    from pyspark import SparkContext
+    from pyspark.ml.util import _jvm
+    from py4j.java_gateway import UserHelpAutoCompletion
+    sc = SparkContext._active_spark_context
+    java_obj = _jvm()
+    for name in java_class.split("."):
```

Comment:
It seems we are making worthless iterations on this loop. Shouldn't be enough just getting `java_obj` of the last element of `java_class.split(".")` ?
i.e. `name = java_class.split(".")[-1]`

Reply:
@danilojsl 
I took this iteration from the PySpark source code, it seems that the way they resolve a classpath in Python and create a object in the JVM
[here]https://github.com/apache/spark/blob/4201a5924b5bd12d2c89f31b6049272748c58728/python/pyspark/ml/wrapper.py#L83  


This looks redundant, but we need to check for all the sub-class-path strings if they exsist.
I.e. if the classpath is `A.B.C.D` we must check 
- `A` 
- `A.B`
- `A.B.C`
- `A.B.C.D`

In python we can have `A.B.C.D` defined, but the other py-class-paths may not be defined and will cause JVM not to find class in PVM